agents/diamagent_test.go

```diff
@@ -259,8 +263,12 @@ func TestProcessRequest(t *testing.T) {
 			}
 			*prply = sessions.V1UpdateSessionReply{
 				Attributes: &engine.AttrSProcessEventReply{
-					MatchedProfiles: []string{"ATTR_1001_SESSIONAUTH"},
-					AlteredFields:   []string{"*req.Password", "*req.PaypalAccount", "*req.RequestType", "*req.LCRProfile"},
+					AlteredFields: []*engine.FieldsAltered{
+						{
+							MatchedProfileID: "ATTR_1001_SESSIONAUTH",
+							AlteredFields:    []string{"*req.Password", "*req.PaypalAccount", "*req.RequestType", "*req.LCRProfile"},
```

Comment:
Please rename AlteredFields here into Fields, as agreed since we have a duplication.